cypress/support/server.ts

```diff
@@ -203,6 +207,45 @@ export const mockGetStorage = (storageAmountInBytes: number): void => {
   ).as('getCurrentMemberStorage');
 };
 
+export const mockGetMemberStorageFiles = (
+  files = MEMBER_STORAGE_ITEM_RESPONSE,
+  shouldThrowError = false,
+): void => {
+  const route = new RegExp(`${API_HOST}/members/current/storage/files`);
+  cy.intercept(
+    {
+      method: HttpMethod.Get,
+      url: route,
+    },
+    ({ url, reply }) => {
+      const params = new URL(url).searchParams;
+
+      const page = parseInt(params.get('page') ?? '1', 10);
```

Comment:
You should not have to specify the radix of the integer conversion. This happens because TS thinks you are using the function from node.js, if you call the function from `window` the issue will disapear:
```suggestion
      const page = window.parseInt(params.get('page') ?? '1');
```